### ISSUE TITLE
why don't implement Logistic regression

### ISSUE THREAD

**Issue by NickYi1990** [automl/auto-sklearn#328]
Hi, dear author
why don't you implement Logistic Regression, is there any difficulties? if it is easy to implement, I would happy to contribute, thanks!

**Comment by mfeurer** [comment]
That's a tricky question. We implement logistic regression via scikit-learn's SGD module (which implements some other loss functions as well). The SGD module has some nice properties like incremental training, but on the other hand does not tune it's learning rate itself like logistic regression does. Do you have any examples/comparisons of the two implementations to help choosing between the two?

**Comment by NickYi1990** [comment]
I think SGD will tune its learning rate itself but Logistic won't~ Em... maybe SGD is good enough and Logistic is not needed.
